Description: Suggest installing the packages that are actually available in Ubuntu.
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Forwarded: no, Ubuntu-specific

Index: virt-manager-0.10.0/virtManager/engine.py
===================================================================
--- virt-manager-0.10.0.orig/virtManager/engine.py	2013-11-29 23:48:02.346071380 +0100
+++ virt-manager-0.10.0/virtManager/engine.py	2013-11-29 23:48:02.342071380 +0100
@@ -182,7 +182,7 @@
         # Manager fail message
         msg = _("Could not detect a default hypervisor. Make\n"
                 "sure the appropriate virtualization packages\n"
-                "are installed (kvm, qemu, libvirt, etc.), and\n"
+                "are installed (qemu-system, libvirt-bin), and\n"
                 "that libvirtd is running.\n\n"
                 "A hypervisor connection can be manually\n"
                 "added via File->Add Connection")
